opium_kernel/src/opium_kernel.mli

```diff
@@ -213,6 +213,24 @@ module Server_connection : sig
     -> 'a Lwt.t
 end
 
+module Static : sig
+  val serve
```

Comment:
not sure why we need to expose this in the mli

Reply:
The static module, as implemented here, can be used in two ways:

- As a middleware, that will read the file with the given filename (the function `m`)
- As a `Rock.Handler` (the function `serve`)

I had a use case for the second point, where I wanted to expose a file with a custom URL.